src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.py_in

```diff
@@ -230,7 +231,7 @@ class FitMultipleModel():
                 images_per_seg,
```

Comment:
On line 223 (call to compute_loss_and_metrics(), metric_eval_time is an unused local var now, so we could just replace it with _ 

(Unimportant, but  I think that's what we've done most other places.)